src/server/ApiRoutes/ApiRoutes.js

```diff
@@ -69,7 +69,8 @@ router
 
 router
   .route('/api/newHold')
-  .get(Hold.createHoldRequestAjax);
+  .get(Hold.createHoldRequestAjax)
+  .post(Hold.createHoldRequestAjaxEdd);
```

Comment:
A `post` is server side and not involved with ajax, so remove `ajax` from the function name.